src/components/Header/Header.jsx

```diff
@@ -2,12 +2,17 @@ import "./Header.css";
 import "../../App.css";
 import { Link, useNavigate } from "react-router-dom";
 import { ResponsiveNavbarForMobile, ResponsiveNavbarForTablet } from "../index";
-import { useNavbar, useAuth } from "../../contexts/index";
+import { useNavbar, useAuth, useWishlist, useCart } from "../../contexts/index";
```

Comment:
If you have used index.js for exporting the data. then while importing it you don't need to specify `/index` it will always take it from index by default. you can simply write `../../contexts` thats it.

Reply:
oh, didnt know that thanks